frontend/src/components/modal/mobile/MobileModal.jsx

```diff
@@ -0,0 +1,39 @@
+import React, { useEffect } from 'react';
+import MobileNav from './nav/MobileNav';
+import PhotosGrid from './photo/PhotosGrid';
+import FadeIn from './styled-components/FadeIn';
+import MobileModalContainer from './styled-components/MobileModalContainer';
+import { useDispatch, useSelector } from 'react-redux';
+import { handleGroupPhotos } from '../../../redux/slices/photos/PhotoSlice';
+
+const MobileModal = () => {
+	const dispatch = useDispatch();
+	const { modal, photos } = useSelector((state) => state);
+
+	useEffect(() => {
+		dispatch(handleGroupPhotos(photos.allPhotos));
+	}, [photos.allPhotos]);
+
+	return (
+		<MobileModalContainer
+			className={'find-em'}
```

Comment:
@manedurphy I'm seeing that you refactored your original App.jsx code for responsiveness to use different components instead of using the conditional! It definitely looks cleaner on the App.jsx page though not really clear on hows it's working within the component, I couldn't find the 'find-em' class in the .css doc, though I see that the style is setting the slide transition settings.

Reply:
Good looking out! If I remember correctly, that "find-em" class was just a dummy class meant for trial-and-error styling. I will remove it in my most recent PR. As for how the responsiveness is working now, it is all handled with media queries in various styled components.